src/SortingAlgorithms/README.md

```diff
@@ -0,0 +1,10 @@
+# Sorting Algorithms
+
+A Sorting Algorithm is used to rearrange a given array or list elements according to a comparison operator on the elements. The comparison operator is used to decide the new order of element in the respective data structure.
+
+-------------
+
+## Topics Covered
+
+- Quick Sort
```

Comment:
Do one thing, link this to the QuickSort.md page

[Quick Sort](relative/path/to/QuickSort.md)

Reply:
I have linked them.